company/okrs/2020_q1.md

```diff
@@ -0,0 +1,107 @@
+   1. Generate at least [$M][M] in net new ARR from existing accounts.
+   1. Hire a Head of Customer Success who can own this function.
+   1. Reach monthly NPS of 60 at all Tier 1 customers.
+
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: Create a customer driven product process**
+   1. All product engineering work has a delivery plan (where "delivery plan" links to a new handbook page that defines it, based on [RFC 97](https://docs.google.com/document/d/1cZ7JIVuRWrF2MxwDdH36SC7zOyT2qJf9AMUd9Wc9_aY/edit#heading=h.dkow31it1qt1)).
+   1. Handbook describes the lifecycle of a feature and who is responsible for each part of the lifecycle.
+   1. Handbook describes product planning processes and API between product and engineering.
+1. **Product: Build the product and design team to support engineering and ARR growth**
+   1. Hire +1 UX Designer.
+   1. Hire +1 Product Manager.
+
+## Engineering
+
+### VP Eng
+
+1. **Build a world class engineering team.**
+   1. Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **Create a data driven planning process.**
+   1. Each team commits to a written work plan for each release cycle.
+   1. Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+### Distribution
+
+1. **Resolve issues for major customers**
+   1. Ensure that no technical issue blocks a deal for any tier 1 or tier 2 customer.
+1. **Improve developer productivity**
+   1. Automate away release captain role and adopt continuous releases (no human in loop required by default).
+   1. e2e and regression test suite is 100% reliable (no flakiness) when run in CI.
+1. **Understand where customers falloff during setup process**
+   1. Time to important admin setup milestones (i.e. instance started, admin account created, >1 repositories added) are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
+   1. 50th and 90th percentile time from account creation to important user onboarding milestones (i.e. first search, first hover tooltip, first go-to-definition, first find references) are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
+
+### Code intelligence
+
+1. **LSIF is adopted by customers**
+   1. At least three customers have > 50% of code intel queries powered by LSIF (excluding fallback references)
+   1. At least one customer has > 75% of code intel queries powered by LSIF (excluding fallback references)
```

Comment:
@efritz I would like to see a more aggressive % here mitigated by the fact that denominator only includes code intel requests for languages that we have an indexer for (I think we would need to tweak/add a metric in your rfc).
```suggestion
   1. At least one customer has > 95% of code intel queries on languages that we have an LSIF generator for powered by LSIF (excluding fallback references).
```

Reply:
> on languages that we have an LSIF generator for

We can't capture that unless we can return language stats in pings.

Reply:
Eh, I think the tracking for my suggestion above would annoyingly explode the data that we would need to collection. How about we just update the % to be more aggressive :)

```suggestion
   1. At least one customer has > 90% of code intel queries powered by LSIF (excluding fallback references)
```

Reply:
> We can't capture that unless we can return language stats in pings.

We could capture it without knowing language stats (client hard codes which languages we have lsif generator for and then we capture aggregate counts of fuzzy code intel requests for only those language), but as I said above I don't think that extra tracking is worth the effort or complexity.